src/index.js

```diff
@@ -290,13 +304,11 @@ function extractRawLocation(item) {
     seekingMoney: extractSeekingMoney(item),
     seekingMoneyURL: extractSeekingMoneyURL(item),
     currentlyOpenForDistributing: item.gsx$currentlyopenfordistributing.$t,
-    openingForDistributingDontations: item.gsx$openingfordistributingdonations.$t,
-    closingForDistributingDonations: item.gsx$closingfordistributingdonations.$t,
+    distributionHours: getHours(item.gsx$openingfordistributingdonations.$t, item.gsx$closingfordistributingdonations.$t),
```

Comment:
```suggestion
    aidDistributionHours: getHours(item.gsx$openingfordistributingdonations.$t, item.gsx$closingfordistributingdonations.$t),
```

Reply:
I also tinkered with `Hours for Distributing Donations` / `Hours for Receiving Donations`

Reply:
This is kind of borders on a bike shed (although it's definitely important), so as the author of the PR I think you should do what you feel is best. We can change the copy text in the translation file later.

Reply:
I think what you've got is good Alyiri,  and if we need to in the future we will try to get feedback from users on that!

Reply:
Tiny side note - I think if you do `hoursForDistributingDonations` as the property name, you'll get `Hours For Distributing Donations` (capital `For`) due to how the props are transformed.